src/components/views/PlayWhites.js

```diff
@@ -0,0 +1,57 @@
+import React, {useEffect, useState} from 'react';
+import {useHistory} from 'react-router-dom';
+import BaseContainer from "components/ui/BaseContainer";
+import "styles/views/WhiteCardSelection.scss";
+import Header from "./Header";
+import CardButton from "../ui/CardButton";
+import MenuContainer from "../ui/MenuContainer";
+
+
+const PlayWhites = () => {
+  // use react-router-dom's hook to access the history
+  const history = useHistory();
+
+  // define a state variable (using the state hook).
+  // if this variable changes, the component will re-render, but the variable will
+  // keep its value throughout render cycles.
+  // a component can have as many state variables as you like.
+  // more information can be found under https://reactjs.org/docs/hooks-state.html
+
+  return (
+    <React.Fragment>
+      <Header view="game"/>
+      <h1>Pick a white card</h1>
+
+      <MenuContainer className={"menu container"}>
```

Comment:
Menu Container is identical to Base Container. Maybe we should just use BaseContainer and then adapt style depending on className

Reply:
Also, I would add more Containers here (one for the whole content, one for the black card to vote one, one for the whitecards)

Reply:
good idea, having fewer containers is probably better to not lose track. This was just to have a first draft ready for the first week. i expect this to change at some point, especially since the functionality still has to be added. but i would merge this now instead of having this branch dangle and getting outdated until we can add the functionality